src/scaladoc/scala/tools/nsc/doc/Uncompilable.scala

```diff
@@ -23,22 +23,39 @@ trait Uncompilable {
   val global: Global
   val settings: Settings
 
-  import global.{ reporter, inform, newTypeName, newTermName, runReporting, Symbol, DocComment, NoSymbol }
-  import global.definitions.AnyRefClass
+  import global.{ reporter, inform, newTypeName, newTermName, runReporting, DefDef, Symbol, DocComment, MethodSymbol, NoSymbol }
+  import global.definitions.{ AnyRefClass, AnyRefTpe }
   import global.rootMirror.RootClass
 
   private implicit def translateName(name: Global#Name) =
     if (name.isTypeName) newTypeName("" + name) else newTermName("" + name)
 
-  def docSymbol(p: DocParser.Parsed) = p.nameChain.foldLeft(RootClass: Symbol)(_.tpe member _)
+  val WaitNames = List("scala", "AnyRef", "wait")
+
+  def docSymbol(p: DocParser.Parsed) =
+    p.nameChain match {
+      // for scala.AnyRef.wait, pick Object#wait with the same arity
+      case xs if xs.map(_.toString) == WaitNames =>
+        val arity = p.docDef.definition match {
+          case t: DocParser#DefDef => t.vparamss.flatten.size
+          case _ => 0
+        }
+        val wait = AnyRefTpe.members find {
+          case m: MethodSymbol =>
+            m.name == newTermName("wait") && m.paramLists.flatten.size == arity
```

Comment:
maybe still add a `case _ => false` (even if members of `AnyRef` are unlikely to change)

Reply:
Addressed in 9f685f78f5373cd98c6ea22873402ee613de1542

Reply:
👍 thanks!